app/controllers/CacheController.scala

```diff
@@ -145,16 +143,14 @@ class CacheController @Inject() (
     }
 
   private def getUserAnswers[T](lrn: String)(f: UserAnswers => T)(implicit writes: Writes[T]): Action[AnyContent] =
-    (authenticate() andThen getVersion).async {
+    authenticate().async {
       implicit request =>
         val eoriNumber = request.eoriNumber
         cacheRepository
           .get(lrn, eoriNumber)
           .map {
-            case Some(userAnswers) if request.phase.isTransitional == userAnswers.isTransitional =>
-              Ok(Json.toJson(f(userAnswers)))
             case Some(userAnswers) =>
-              BadRequest
+              Ok(Json.toJson(f(userAnswers)))
```

Comment:
Think the removed case still applies. We don't want users regaining access to their transitional documents, but the logic would now become
```
case Some(userAnswers) if userAnswers.isTransitional =>
  BadRequest
case Some(userAnswers) =>
  Ok(Json.toJson(f(userAnswers)))
```